cloud.ks

```diff
@@ -0,0 +1,128 @@
+text # don't use cmdline -- https://github.com/rhinstaller/anaconda/issues/931
```

Comment:
question :), so I am guessing when running the pipeline job, this ks file will be visible on the node too? What is the reasoning behind that? Did jenkins use the checkedout repo location as its working directory by any chance? (/me asking with little knowledge here, so forgive my ignorance :p)

Reply:
You're exactly right! Jenkins will checkout the code into the current directory.